lib/media.js

```diff
@@ -97,10 +97,16 @@ exports.setTrack = function (item, callback) {
       callback(err)
       return
     }
-    var duration = 0
+    var duration;
```

Comment:
> not sure why removing initial value?
> also please be consistent without semicolons.

I felt there was no need to set the value since the parser method initialises the value to 0 again.

Also my bad about the semicolons. Will work on improving this PR on Monday.